Directory.Build.props

```diff
@@ -25,6 +25,13 @@
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <DeveloperBuildTestTfms>netcoreapp2.1</DeveloperBuildTestTfms>
+    <StandardTestTfms>$(DeveloperBuildTestTfms)</StandardTestTfms>
+    <StandardTestTfms Condition=" '$(DeveloperBuild)' != 'true' ">netcoreapp2.1;netcoreapp2.0</StandardTestTfms>
```

Comment:
Any reason 2.0 is still here?

Reply:
I'm trying to keep this PR strictly about source code organization, and not changing the contents of said code unless necessary.

Reply:
OK I guess we can do some other pass later.